pkg/config/plugins/runtime/k8s/config.go

```diff
@@ -1,17 +1,63 @@
 package k8s
 
 import (
+	"net/url"
+	"time"
+
 	"github.com/pkg/errors"
+	"go.uber.org/multierr"
+	kube_api "k8s.io/apimachinery/pkg/api/resource"
 
 	"github.com/Kong/kuma/pkg/config"
 )
 
 func DefaultKubernetesRuntimeConfig() *KubernetesRuntimeConfig {
 	return &KubernetesRuntimeConfig{
-		CNIEnabled: false,
 		AdmissionServer: AdmissionServerConfig{
-			Address: "", // all addresses
-			Port:    5443,
+			Port: 5443,
+		},
+		Injector: Injector{
+			CNIEnabled: false,
+			ControlPlane: ControlPlane{
+				ApiServer: ApiServer{
```

Comment:
This property is not needed - it should be taken from other parts of kuma-cp config (I think catalog)